### DISCUSSION TITLE
[Keyball46保守部品]_はんだ付け済トラックボール読み取り基板 の互換性について

### DISCUSSION THREAD

**Discussion by shiro30330** [Yowkees/keyball#115] [Q&A] [answered]
Keyball39 を組み立て中です。

トラックボールが動作しないため、https://github.com/Yowkees/keyball/discussions/51#discussioncomment-1044412 で挙げていただいている 1. ~ 5. を確認してみたのですが改善されなかったのでトラックボール読み取り基板ごと取り替えたく [こちら](https://yowkees.booth.pm/items/3146963) の購入を検討しています。

それにあたり 2 点質問させてください。

1. Keyball39 にも使えますでしょうか
1. もし使えるのであれば購入したいのですが 次の入荷 (作成?) はいつ頃になりますでしょうか

お忙しいところ恐縮ですがご回答いただけると幸いです。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。了解しました。今在庫を追加しますので、見てください。
（以前、Keyball購入者でないっぽい方から大量の注文が来てしまい、在庫を落としておりました。）
半導体価格が高く、ちょっと高額になってしまっており申し訳ありません。動作確認をした上でお送りしますので、まず大丈夫と思いますが、それでも動かないときはご連絡ください。

**Comment by Yowkees** [comment]
回答し忘れましたが、Keyball39でも使用可能です。よろしくお願いいたします。

**Comment by shiro30330** [comment]
迅速にご回答いただきありがとうございます！

購入させていただきました、ありがとうございます！

**Comment by shiro30330** [comment]
さきほど読み取り基板が届きまして組み合わせてみたところ無事トラックボールが動作しました！

ありがとうございます！

**Reply by Yowkees** [reply]
無事に動いて良かったです！ありがとうございました！m(_ _)m
